telco-examples/edge-clusters/dhcp/eib/telco-edge-cluster.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: 1.0
+image:
+  imageType: RAW
+  arch: x86_64
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-RT-GM.raw
+  outputImageName: eibimage-slemicro55rt-telco.raw
+operatingSystem:
+  rawConfiguration:
+    diskSize: 40G
+  time:
+    timezone: America/New_York
+  kernelArgs:
+    - ignition.platform.id=openstack
+    - net.ifnames=1
+  systemd:
+    disable:
+      - rebootmgr
+  users:
+    - username: root
+      encryptedPassword: ${ROOT_PASSWORD}
+  packages:
+    packageList:
+      - jq
+      - dpdk22
+      - dpdk22-tools
+      - libdpdk-23
+      - pf-bb-config
+    additionalRepos:
+      - url: https://download.opensuse.org/repositories/isv:/SUSE:/Edge:/Telco/SLEMicro5.5/
+    sccRegistrationCode: ${SCC_REGISTRATION_CODE}
```

Comment:
you need to append the preprovisioningNetworkDataName parameter and add a secret

Reply:
this is the telco-edge-cluster definition for EIB... I don't need this IMHO

Reply:
This is the definition for the EIB I don't think I need it here